kong/clustering/services/sync/rpc.lua

```diff
@@ -54,13 +55,29 @@ local function full_sync_result()
 end
 
 
+local function get_current_version()
+  return declarative.get_current_hash() or DECLARATIVE_EMPTY_CONFIG_HASH
+end
+
+
+local is_valid_version
+do
+  local VER_PREFIX = "V02_"
```

Comment:
```suggestion
  local VER_PREFIX = "v02_"
```

I still prefer lowercased `v`, this looks like the format others use (e.g. GitHub tokens).

Reply:
done.